Docs: Fix gpu-arch-spec.rst (#4393)
ROCm/ROCm: docs/reference/gpu-arch-specs.rst

Size of the Scalar General Purpose Register (SGPR) file. Holds data used in
scalar instructions.

**GFXIP**

GFXIP (Graphics IP) is a versioning system used by AMD to identify the GPU
architecture and its instruction set. It helps categorize different generations
of GPUs and their feature sets.

**GFXIP major version**

> Defines the core instruction set and architecture of the GPU, which determines
> compatibility with software stacks such as HIP and OpenCL. For example, a GFXIP

Reviewer: ```suggestion
Defines the GPU's core instruction set and architecture, which determines
compatibility with software stacks such as HIP and OpenCL. For example, a GFXIP
```

> major version of 11 corresponds to the RDNA 3 (Navi 3x) architecture, influencing

Reviewer: ```suggestion
11 major version corresponds to the RDNA 3 (Navi 3x) architecture, influencing
```

driver support and available compute features.

**GFXIP minor version**

Represents specific variations within a GFXIP major version, affecting feature sets,

> optimizations, and driver behavior in software stacks like HIP and OpenCL. Different

Author: Fixed

> GPU models within the same major version may have unique capabilities, impacting

Reviewer: ```suggestion
GPU models within the same major version can have unique capabilities, impacting
```

Author: Fixed
